package.json

```diff
@@ -13,22 +13,31 @@
   "devDependencies": {
     "@types/cors": "2.8.14",
     "@types/express": "4.17.19",
+    "@types/jest": "^29.5.13",
     "@types/node": "20.8.6",
+    "@types/supertest": "^6.0.2",
     "@types/swagger-jsdoc": "6.0.1",
     "@types/swagger-ui-express": "4.1.4",
     "@typescript-eslint/eslint-plugin": "6.7.5",
     "@typescript-eslint/parser": "6.7.5",
     "eslint": "8.51.0",
     "eslint-config-prettier": "9.0.0",
     "eslint-plugin-prettier": "5.0.1",
+    "jest": "^29.7.0",
```

Comment:
Nos packages instalados, você verá as nomenclaturas ^, ~, etc.
O ^ significa que quando rodar o install, ele puxará da versão estipulada até a última versão disponível.
ou seja, se tiver `"jest": "^29.7.0",`, ele poderá instalar do 29.7 até a última versão disponível. se no ano que vem lançarem o 30.0, ele instalaria ela. 

Isso traz alguns riscos para o projeto, pense que você precisa subir esse projeto ano que vem, quando deu o install, ele puxou a versão 30.0 e nessa versão, eles mudaram a forma de configuração do jest. O seu projeto não rodaria.  O Package-lock, mantém o histórico das versões das bibliotecas instaladas, porém é comum de se ter alguns problemas em instalações de bibliotecas e remover o package-lock e o node-modules e reinstalar as bibliotecas. Ou seja, isso colocaria em risco o seu projeto.

Para garantirmos que o projeto estará funcionando devidamente, independente de quando rodarmos, é importante nós fixarmos as bibliotecas, sem a utilização dos acrônimos de majors e minors ^, ~, pois assim garantimos a instalação da versão que desejamos em todos os ambientes e atualizamos elas quando for necessário.

Posteriormente, entraremos mais a fundo sobre as versões dos pacotes, manter a segurança deles.

Deixar um link que explica um pouco mais sobre o versionamento semantico
[](https://nodejs.reativa.dev/0024-npm-semantic-versioning/index)

Sendo assim, vamos fixar todas as versões das bibliotecas, para garantir que o projeto de vocês mantenham sempre funcionando. :1st_place_medal: